shoop/core/order_creator/_creator.py

```diff
@@ -22,21 +22,7 @@
 from ._source_modifier import get_order_source_modifier_modules
 
 
-class OrderCreator(object):
-
-    def __init__(self, request=None):
-        """
-        Initialize order creator.
-
-        :type request: django.http.HttpRequest|None
-        :param request:
-          Optional request object for backward compatibility.  Passing
-          non-None value is DEPRECATED.
-        """
-        if request is not None:
-            warnings.warn(
-                "Initializing OrderCreator with a request is deprecated",
-                DeprecationWarning, stacklevel=2)
+class OrderProcessor(object):
```

Comment:
@suutari-ai said that same file is good for now because we don't want to refactor too much. I agree with that.